stellar-dotnet-sdk/LedgerKeyClaimableBalance.cs

```diff
@@ -16,12 +16,12 @@ public LedgerKeyClaimableBalance(byte[] balanceId)
     }
 
     /// <summary>
-    /// Constructs a <c>LedgerKeyClaimableBalance</c> object from a base64-encoded string. 
+    /// Constructs a <c>LedgerKeyClaimableBalance</c> object from a hex-encoded string. 
     /// </summary>
-    /// <param name="balanceId"></param>
+    /// <param name="balanceId">Hex-encoded string of a claimable balance ID.</param>
     public LedgerKeyClaimableBalance(string balanceId)
     {
-        BalanceId = Convert.FromBase64String(balanceId);
+        BalanceId = Convert.FromHexString(balanceId);
```

Comment:
Breaking change?

Reply:
No, this constructor has only been added in this PR.